components/stepper/overview.md

```diff
@@ -53,18 +48,54 @@ To use a Telerik Stepper for Blazor:
 }
 ````
 
-## Features
+## Steps
+
+The Steps are the building blocks of the Stepper component. You can include the desired steps by adding a `StepperStep` tag for every step. The steps appearance can be configured through the features the `StepperStep` exposes. [Read more about the Blazor Stepper configuration]({%slug stepper-steps-overview%}).
+
+## Orientation
+The Stepper component provides different orientations. [Read more about the Blazor Stepper orientation]({%slug stepper-orientation%}).
+
+## Display Modes
+
+The Stepper component provides different display modes. [Read more about the Blazor Stepper display modes]({%slug stepper-display-modes%}).
+
+## Linear Flow
 
-The Stepper provides the following features:
+By default, the user can select any step in the Stepper component. This behavior could be customized by configuring its Linear flow property. [Read more about the Blazor Stepper linear flow]({%slug stepper-linear-flow%}).
```

Comment:
```suggestion
By default, the user can select any step in the Stepper component and go to it directly. This behavior can be changed with the `Linear` parameter, so that the user goes through the steps sequentially. [Read more about the Blazor Stepper linear flow]({%slug stepper-linear-flow%}).
```